### PR TITLE
Add `install-path` action argument to customise where minikube executables get installed

### PR DESCRIPTION
**Pull request by dfr-exnaton**
This PR adds an optional `bin-path` action argument such that the directory where minikube gets installed can be configured. This is useful for self-hosted runners where the hardcoded `/home/runner` doesn't necessarily exists.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -212,6 +212,14 @@ By default setup-minikube caches the ISO, kicbase, and preload using GitHub Acti
   </pre>
 </details>
 
+<details>
+  <summary>bin-path (optional)</summary>
```
Comment (medyagh) [inline README.md:216]: thank you for this PR, how about a more descriptive name, such as "install-path" ?

**Review by medyagh — CHANGES_REQUESTED** [review]

**Comment by medyagh** [comment]
@dfr-exnaton are you still interested in this PR ?

File: README.md (hunk 1 above)
Comment (dfr-exnaton) [reply README.md]: Hey, sorry - I oversaw the notification. Thanks for the suggestion, sounds better (I think I just took over the existing binPath variable name and exposed it...).  [ad9137a](https://github.com/medyagh/setup-minikube/pull/214/commits/ad9137aefa65986c0f2a954394ba4ac61afd8f7d) renames the arg to `install-path`.

**Comment by dfr-exnaton** [comment]
> @dfr-exnaton are you still interested in this PR ?

@medyagh friendly ping in case you didn't see my feedback - is there anything preventing this from getting merged?

**Comment by dfr-exnaton** [comment]
@medyagh another friendly ping (I know how it is :D)

**Review by medyagh — APPROVED** [review]

**Comment by medyagh** [comment]
@dfr-exnaton I am  curious why the tests are not running on it

**Comment by medyagh** [comment]
> Just did - now the 2 workflows seem to require your approval (see https://docs.github.com/de/actions/managing-workflow-runs/approving-workflow-runs-from-public-forks)?

before this it wasnot showing the approval request, now it shows...strange

**Comment by dfr-exnaton** [comment]
As you pointed out, I think it was the missing pull from upstream that solved it. Tests seem to pass, so should look good to merge (only you have the permission!)?

**Comment by medyagh** [comment]
@dfr-exnaton thank you for the contribution and this will be included in the next version

**Pull request merged**